website/docs/cli/commands/validate.html.md

```diff
@@ -153,3 +188,17 @@ exact positions used for particular error messages are intended for human
 interpretation only and subject to change in future versions of Terraform due
 either to improvements to the error reporting or changes in implementation
 details of the language parser/evaluator.
+
+### Expression Value
+
+An expression value object gives additional information about a value which is
+part of the expression which triggered the diagnostic. This is especially
+useful when using `for_each` or similar constructs, in order to identify
+exactly which values are responsible for an error. The object has two properties:
+
+* `traversal` (string): An HCL traversal string, such as `var.instance_count`.
```

Comment:
It might not actually matter in practice, but I'm not 100% sure that what we include here is necessarily a valid traversal that HCL could parse, because I think we end up populating it from an `addrs.Reference` rather than from a `hcl.Traversal` and typically our `String` methods in `addrs` aim to produce something Terraform-syntax-_like_ rather than necessarily something exact grammatical.

If you know from working on the code here that I'm wrong about the above then no worries, but otherwise I might suggest to weaken this to also say it's human-readable and subject to change in future, just so we're not over-promising here.